instrumentation/README.md

```diff
@@ -58,6 +58,31 @@ Java instrumentation jar. Typically, it will be set to something like:
 
 to set the deployment environment for the Splunk backend.
 
+### disable_telemetry (optional)
+
+Set this value to `true` to disable the preloader from sending the `splunk.linux-autoinstr.executions` metric to the
+local collector. Default: `false`.
+
+### generate_service_name (optional)
+
+Set this value to `false` to prevent the preloader from setting the `OTEL_SERVICE_NAME` environment variable.
+Default: `true`.
```

Comment:
Is there a doc or brief description you could provide of what the `false` behavior is?

Reply:
Which takes precedence if both `generate_service_name` is `true` and `service_name` is set?

Reply:
Turns out there really isn't a doc yet that describes in detail what the instrumentation library does, although there is [this](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/resource-providers). But I've added a couple of sentences to explain the implications of the false behavior.

Regarding precedence, `generate_service_name=true` will cause the .so generate the service name either dynamically or via the `service_name` override. If it's false, no service name will be set, regardless of the value of the `service_name` override. I've added an explanation to the readme as well as a test case.